app/pages/index.html

```diff
@@ -29,5 +29,6 @@
   <script src="/scripts/app.js"></script>
   <script src="/scripts/services/Room.js"></script>
   <script src="/scripts/controllers/RoomsCtrl.js"></script>
+  <script src="/node_modules/angular-ui-bootstrap/src/modal/modal.js"></script>
```

Comment:
This should work fine. `npm` will install the modules into the `node_modules` directory just like you have them included.